Teensy/src/gpio/gpio.cpp

```diff
@@ -50,7 +50,31 @@ static uint32_t gpio_getEngineRevs()
 
 static int32_t gpio_getDamperPosition()
 {
-    return 0;
+    // Defining pins //
+    #include <Arduino.h>
+
+    float exten1, exten2, exten3, exten4;
+    float DampPos1, DampPos2, DampPos3, DampPos4;
+
+    // Initialising transmission //
+    Serial.begin(9600);
+
+    // Reading the voltage outputs //
+
+    exten1 = analogRead(PIN_A0);
+    exten2 = analogRead(PIN_A1);
+    exten3 = analogRead(PIN_A2);
+    exten4 = analogRead(PIN_A3);
+
+    // Calibrating outputs to damper position //
+
+    DampPos1 = (64 - exten1)/63000;
```

Comment:
The extern variables aren't needed here, instead, we can just do the following - 
```
damperFrontRight = (64 - analogRead(DAMPER_FR_PIN)) / 63000;
```
Also, variables should have a lower-case first letter